packages/wallets/src/hooks/useSyncLocalStorageClientAccounts.ts

```diff
@@ -42,44 +49,61 @@ const useSyncLocalStorageClientAccounts = () => {
                 setLocalStorageClientAccounts(localStorageData);
             }
         },
-        [data, setLocalStorageClientAccounts, settingsData]
+        [setLocalStorageClientAccounts, settingsData]
```

Comment:
Hi @wojciech-deriv, just curious. Does adding `mutateAsync` to the dependancy array cause any issues here?

```suggestion
        [setLocalStorageClientAccounts, settingsData, mutateAsync]
```